homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -159,7 +164,7 @@ async def async_setup_gateway_entry(
     gateway_model = f"{gateway_info.model}-{gateway_info.hardware_version}"
 
     device_registry = await dr.async_get_registry(hass)
-    device_registry.async_get_or_create(
+    await device_registry.async_get_or_create(
```

Comment:
This method is not async https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/device_registry.py#L243

Reply:
Ill revert that